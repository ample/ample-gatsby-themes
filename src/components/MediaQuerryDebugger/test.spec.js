import React from "react"
import renderer from "react-test-renderer"
import MediaQueryDebugger from "./index"

describe("MediaQueryDebugger", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<MediaQueryDebugger isVisible={'true'}/>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
