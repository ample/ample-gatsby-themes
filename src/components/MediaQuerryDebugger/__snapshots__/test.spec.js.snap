// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MediaQueryDebugger renders correctly 1`] = `null`;
